packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -2,4 +2,142 @@
 // PAGINATIONBAR
 //
 
-// .hds-pagination-bar {}
+@use "../mixins/focus-ring" as *;
+
+// * Pagination Parent (wrapper) component
+// ----------------------------------------
+.hds-pagination-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 36px;
+  color: var(--token-color-foreground-primary);
+  // font-size: var(--token-typography-body-300-font-size); // = 1rem, not same as 13px, TODO: investigate
+  font-size: 13px;
+}
+
+// * Sub-component: Pagination
+// ----------------------------------------
+.hds-pagination {
+  display: flex;
+  align-self: stretch;
+}
+
+.hds-pagination__page-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  > li {
+    display: flex;
+  }
+}
+
+.hds-pagination__page-link,
+.hds-pagination__ellipsis {
+  display: flex;
+  align-items: center;
+  padding: 0 12px;
+}
+
+.hds-pagination__page-link {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: 5px,
+    $right: 5px,
+    $bottom: 5px,
+    $left: 5px,
+  );
+  color: inherit;
+  text-decoration: none;
+
+  &:hover {
+    color: var(--token-color-foreground-action-hover);
+  }
```

Comment:
I think the active state is missing here
`&:active {
    color: var(--token-color-foreground-action-active);
  }`